site/docs/how-tos/operator.md

```diff
@@ -0,0 +1,309 @@
+---
+title: How to install Hazelcast Jet Operator 
```

Comment:
Maybe mention Kubernetes in the title somehow so it's visible from the menu (Install Kubernetes Hazelcast Jet Operator)

Reply:
How to titles shouldn't include "how to" but should be action based